src/Main.java

```diff
@@ -1,5 +1,84 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("home 3");
+
+        int age = 15;
+        if (age >= 18) {
+            System.out.println("Если возраст равен " + age + ", то Вы совершеннолетний");
+        }
+        if (age < 18) {
+            System.out.println("Если возраст равен " + age + ", Вы не достигли совершеннолетия, нужно немного подождать");
+        }
+        //задача 1 окончена
+
+        int temperature = 4;
+        if (temperature > 5) {
+            System.out.println("На улице " + temperature + " градуса" + ", нужно надеть шапку");
+        }
+        if (temperature < 5) {
+            System.out.println("На улице " + temperature + " градуса" + ", можно надеть шапку");
+        }
+        //задача 2 окончена
+
+        int speed = 40;
+        if (speed > 60) {
+            System.out.println("Если скорость выше " + speed + ", то придётся заплатить штраф");
+        }
+        if (speed < 60) {
+            System.out.println("Если скорость ниже " + speed + ", то можно ездить спокойно");
+        }
+        //задача 3 окончена
+
+        int age2 = 21;
+        if (age2 >= 2 && age2 <= 6) {
+            System.out.println("Если возраст человека равен " + age2 + " то ему нужно ходить в садик");
+        } else if (age2 >= 7 && age2 <= 17) {
+            System.out.println("Если возраст человека равен " + age2 + " то ему нужно ходить в школу");
+        } else if (age2 >= 18 && age2 <= 24) {
+            System.out.println("Если возраст человека равен " + age2 + " то ему нужно ходить в университет");
+        } else if (age2 > 24) {
+            System.out.println("Если возраст человека равен " + age2 + " то ему нужно ходить на работу");
+        } else {
+            System.out.println("Вы ввели некорректные данные");
+        }
+        //задача 4 окончена
+
+
+        int age3 = 10;
```

Comment:
зачем создавать новую переменную, когда можно просто переиспользовать созданную выше?
```suggestion
        age = 10;
```